examples/01-Getting-started.ipynb

```diff
@@ -0,0 +1,1012 @@
+{
```

Comment:
<p>The content below is great - but I think that is not our default recommendations, how to use NVTabular.</p><p> </p><p>I think we should:</p><ul><li>store the dataset to disk with</li><li class="ql-indent-1">workflow.fit_transformed(train_ds).to_parquet</li><li class="ql-indent-1">workflow.transform(valid_ds).to_parquet</li><li>load the dataset from disk to review it</li></ul><p> </p><p>MovieLens is a small dataset - if users apply it to realistic dataset, they might see OOM errors. I think calling .compute() is for more advanced users to debug/test pipelines. I think we can explain that in another notebook, but the getting started notebook should be simple</p><p> </p><p>To review it, we can do .head() or other statistics like to show that the fields are continous integers.</p><p> </p><pre class="ql-syntax" spellcheck="false">print('Unique movie ids: ' + str(df['movie_id'].unique().shape[0]))
print('Unique user ids: ' + str(df['user_id'].unique().shape[0]))
print('Movie ids min/max: ' + str(df['movie_id'].min()) + ' ' + str(df['movie_id'].max())) 
print('User ids min/max: ' + str(df['user_id'].min()) + ' ' + str(df['user_id'].max()))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>👍made the change</p>